CLFSWM should depend on sb-posix since src/tools.lisp uses sb-posix:getenv & sb-posix:putenv.
--- a/clfswm.asd
+++ b/clfswm.asd
@@ -78,7 +78,7 @@
 			 (:file "bindings-second-mode"
 				:depends-on ("clfswm" "clfswm-util" "clfswm-query" "bindings" "clfswm-pack" "clfswm-menu" "menu-def"
 						      "clfswm-layout")))))
-  :depends-on ( #-:CLX :clx ))
+  :depends-on ( #-:CLX :clx #+:sbcl :sb-posix ))
 
 
 
